template/test/ansible/playbook.yml

```diff
@@ -2,7 +2,19 @@
 - hosts: all
   become: yes
   tasks:
+    - name: Check that the env.yml file exists
+      stat:
+        path: /vagrant/env.yml
+      register: file
+    - name: Include vars of env.yml into the 'env' variable
+      include_vars:
+        file: /vagrant/env.yml
+        name: env
+      when: file.stat.exists
+
```

Comment:
Should it be possible to override these values in **env.yml** file using **env_override.yml file**?
We do not have any checks to make sure that the env.yml file does contain *nomad_acl* env variable so it will be better if this file holds all the env variables with default values and is not modified.
The matrix tests then could make use of env_override.yml file to override any of the existing default env values.
Your thoughts?

Reply:
Yup, I agree. Also makes it more explicit for those using it